### PR TITLE
Deprecate this buildpack

### PR DESCRIPTION
**Pull request by dzuelke**
That way the list of buildpacks gets exposed through the API, so it's easily visible to customers, support etc.

### CODE REVIEW

**Comment by dmathieu** [comment]
Huge :+1:

**Comment by tt** [comment]
I don't think we should encourage forking a deprecated buildpack.

**Comment by dzuelke** [comment]
Deprecation notice added and README updated accordingly.

File: README.md [hunk 1]
```diff
@@ -1,27 +1,49 @@
...
-    https://github.com/heroku/heroku-buildpack-nodejs.git#0198c71daa8
-    https://github.com/heroku/heroku-buildpack-ruby.git#v86
+Run `heroku help buildpacks` for a full list of commands available to manage buildpacks.
 
 ## License
 
-MIT
+The MIT License (MIT)
+
+Copyright (c) Heroku, Inc.
```
Comment (tt) [inline README.md]: I'm not sure you can add this line when our buildpack is a fork too.

Comment (dzuelke) [inline README.md]: I was wondering the same, but given how the original author was a Heroku employee at that time, I figured it'd just be a clarification, not a change?!

Comment (tt) [inline README.md]: Some contributors are not, though. I think it's easier to just remove that commit. It doesn't make much of a difference if we're going to deprecate it.

**Comment by craigkerstiens** [comment]
:+1:

**Comment by tt** [comment]
Looks good to me.

**Comment by jkutner** [comment]
@dzuelke lgtm. may i merge?

**Comment by dzuelke** [comment]
Go ahead!

**Comment by owenthereal** [comment]
FYI, this buildpack may be still useful for a kernel app.

/cc @mikehale

**Comment by dzuelke** [comment]
Yeah we're just deprecating, not removing anytime soon (and most people are likely running off ddollar's repo anyway).

**Review by ojacobson — APPROVED** [review]
🚢 🇮🇹

**Review by dmathieu — APPROVED** [review]
🚀 🎈 💥

**Pull request merged**